### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by reedy**
Ping #1235

### CODE REVIEW

**Comment by chrisrosset** [comment]
If you're renaming `openlayers.js` maybe we could move it to `/js`?

File: js/jquery.blockUI.js [hunk 1]
```diff
@@ -12,488 +12,547 @@
...
-	var t = ((p.offsetHeight - el.offsetHeight)/2) - sz(p,'borderTopWidth');
-	if (x) s.left = l > 0 ? (l+'px') : '0';
-	if (y) s.top  = t > 0 ? (t+'px') : '0';
-};
-
-function sz(el, p) {
-	return parseInt($.css(el,p))||0;
-};
-
+(function ($) {
```
Comment (chrisrosset) [inline js/jquery.blockUI.js]: I think we should exclude this 3rd party file. We should separate all the 3rd party stuff out at some point.

Comment (reedy) [reply js/jquery.blockUI.js]: Yeah, I was just looking around for a minified version of this...

Which lead to the rabbit hole of fixing the docs - https://github.com/malsup/blockui/pull/162

File: test/server/README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
 This test suite validates PHP calls and MySQL database content.
 
-*** Instructions
+\*\*\* Instructions
```
Comment (chrisrosset) [inline test/server/README.md]: This looks weird - is this definitely right?

File: sql/README [hunk 3]
```diff
@@ -1,23 +1,23 @@
-*** Tools for users
+\*\*\* Tools for users
```
Comment (chrisrosset) [inline sql/README]: The changes in this file should be undone - it's plaintext, not markdown, and the indents were used on purpose.

File: test/server/README.md (hunk 2 above)
Comment (reedy) [reply test/server/README.md]: The very opiniated prettier thinks so ;)

Comment (chrisrosset) [reply test/server/README.md]: I've never seen it do this. Let me take a quick look.

Comment (chrisrosset) [reply test/server/README.md]: Note that the main README.md file does not suffer the same "fix."

Comment (chrisrosset) [reply test/server/README.md]: OK, the original is wrong - I thought it was hashes.

```suggestion
### Instructions
```

File: sql/README (hunk 3 above)
Comment (reedy) [reply sql/README]: I wonder if we should make it MD for consistency...

**Comment by reedy** [comment]
Dropped `effects.js` too as unused. Probably should've gone with eb830a82787939a1ca999b0b406da88f82dc8280

**Comment by reedy** [comment]
Probably GTG at this point...

CI running correctly for direct push would be nice, but that can happen later...

File: sql/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+### Tools for users
+
+- [`create.sql`](create.sql) - Create empty OpenFlights database schema
+- [`export.sql`](export.sql) - Export airlines.dat, airports.dat and routes.dat from OpenFlights database
```
Comment (reedy) [inline sql/README.md]: This disappeared in 4ff8baf9d3daf43c8890cac9b08846a787fde420

**Comment by reedy** [comment]
Any objects to merging this at this point?

**Comment by chrisrosset** [comment]
> Any objects to merging this at this point?

This is great! Let's gooooooooooo.

**Pull request merged**